### PR TITLE
Move `RecoTracker/PixelVertexFinding` to `RecoVertex/PixelVertexFinding`

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Title says it all, as promised at https://github.com/cms-sw/cmssw/pull/41223#issuecomment-2277561265.
Purely technical, no regressions expected

#### PR validation:

Tested:
   * unit tests: `scram b runtests`
   * `addOnTests.py`
   * `runTheMatrix.py -l limited --ibeos`

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Not a backport, no backport needed.

### CODE REVIEW

File: RecoVertex/PixelVertexFinding/python/PixelVertexes_cfi.py
Comment (fwyzard) [inline RecoVertex/PixelVertexFinding/python/PixelVertexes_cfi.py:1]: Taking the opportunity of the move, could you also rename it to `PixelVertices_cfi.py` to match the label of the module ?

Comment (mmusich) [reply RecoVertex/PixelVertexFinding/python/PixelVertexes_cfi.py:1]: sure.

Comment (mmusich) [reply RecoVertex/PixelVertexFinding/python/PixelVertexes_cfi.py:1]: Done at https://github.com/cms-sw/cmssw/pull/45842/commits/243a0405cbcd6c8a0da45119fd9785a118bb362d

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by perrotta** [comment]
+alca

**Comment by mmusich** [comment]
@cmsbuild ping

**Comment by jfernan2** [comment]
+1

**Comment by mmusich** [comment]
@cms-sw/dqm-l2 kind ping

**Comment by tjavaid** [comment]
+1

- P5 tests OK.

**Comment by mmusich** [comment]
@cms-sw/orp-l2 can you merge this PR? Soon there will be many conflicts to other PRs changing same code.

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**